### PR TITLE
Balance for Loyalists 1.18

### PR DESCRIPTION
**Pull request by Hejnewar**
Changes:
Level 1:
Cavalryman - blade resistance changed from 30% to 20%, xp changed from 40 to 44.
Bowman - xp changed from 39 to 35, melee damage changed from 4 to 5.
Horseman - cost changed from 23 to 24, xp changed from 44 to 54. 
Merman - xp changed from 36 to 34. 
Mage - xp changed from 54 to 48.

Level 2: 
Javelineer - ranged damage increased from 11 to 13, cost changed from 25 to 24. 
Pikeman - cost changed from 25 to 24, xp changed from 65 to 69.
Swordsman - xp changed from 60 to 80.
Knight - cost changed from 38 to 56, but I'm inclined to leave it at 50 to the fun of AoH WC players, xp changed from 120 to 100. 
Lancer - cost changed from 44 to 47. 
Shock Trooper - cost changed from 35 to 28, xp changed from 110 to 65.
Red mage - cost changed from 38 to 35, xp changed from 100 to 83.
White mage - cost changed from 38 to 31, xp changed from 136 to 75.
Dragoon - cost changed from 28 to 29, xp changed from 95 to 80.
Longbowman - cost changed from 26 to 23, xp changed from 68 to 65. 
Duelist - xp changed from 90 to 80, price changed form 29 to 31.
Merman Warrior - cost changed from 26 to 24, xp changed from 80 to 60.
Lieutenant - cost changed from 34 to 31, xp changed from 80 to 85.

Level 3:
Halberdier - cost changed from 44 to 52.
Royal Guard - cost changed from 44 to 61
GK - hp changed from 78 to 84, lance damage changed from 17 to 19, cost changed from 58 to 86.
Paladin - hp changed from 65 to 77, lance damage changed from 15 to 16, melee damage changed from 8 to 9, cost changed from 58 to 82.
Iron Mauler cost changed from 50 to 49.
Archmage - cost changed from 59 to 63 xp changed from 220 to 150.
Mage of Light - cost changed from 60 to 59.
Cavalier - cost changed from 52 to 62.
Master Bowman - cost changed from 46 to 50.
Master at Arms - melee damage increased from 7 to 8, cost changed from 44 to 65.
Hoplite - cost changed from 43 to 45.
Triton - cost changed from 43 to 46.
General - hp changed from 50 to 55, cost changed from 54 to 66, xp changed from 180 to 105.

Level 4: 
Great Mage - ranged damage changed from 16 to 17, cost changed from 72 to 110. 
Grand Marshal - hp changed from 60 to 68 cost changed from 66 to 101.

There are still few units that didnt make it here (HI, Silver Mage, Elder Mage).

### CODE REVIEW

File: data/core/units/humans/Horse_Paladin.cfg [hunk 1]
```diff
@@ -6,7 +6,7 @@
     image="units/human-loyalists/paladin/paladin.png"
     image_icon="units/human-loyalists/paladin/paladin.png~CROP(8,10,72,72)"
     profile="portraits/humans/paladin.webp"
-    hitpoints=65
+    hitpoints=6775
```
Comment (knyghtmare) [inline data/core/units/humans/Horse_Paladin.cfg:9]: @Hejnewar Bruh...XD..I mean what?

File: data/core/units/humans/Loyalist_Grand_Marshal.cfg [hunk 2]
```diff
@@ -11,15 +11,15 @@
             image="units/human-loyalists/marshal-leading.png:300"
         [/frame]
     [/leading_anim]
-    hitpoints=60
+    hitpoints=68
```
Comment (knyghtmare) [inline data/core/units/humans/Loyalist_Grand_Marshal.cfg:14]: Thank you for undoing the injustice the poor GM suffered

File: data/core/units/humans/Loyalist_General.cfg [hunk 3]
```diff
@@ -17,15 +17,15 @@
             halo_x,halo_y=-8,-30
         [/halo_frame]
     [/leading_anim]
-    hitpoints=50
+    hitpoints=55
```
Comment (knyghtmare) [inline data/core/units/humans/Loyalist_General.cfg:20]: Finally

File: data/core/units/humans/Horse_Paladin.cfg (hunk 1 above)
Comment (Hejnewar) [reply data/core/units/humans/Horse_Paladin.cfg:9]: Fair and Balanced as all things should be.

File: data/core/units/humans/Loyalist_Cavalryman.cfg [hunk 4]
```diff
@@ -44,7 +44,6 @@ Cavalrymen are very useful for taking and holding positions on open ground, for
     [/movement_anim]
     {DEFENSE_ANIM "units/human-loyalists/cavalryman/cavalryman-defend2.png" "units/human-loyalists/cavalryman/cavalryman-defend1.png" {SOUND_LIST:HORSE_HIT} }
     [resistance]
-        blade=70
```
Comment (cooljeanius) [inline data/core/units/humans/Loyalist_Cavalryman.cfg:47]: I always get confused by resistances; remind me if this is a buff or a nerf?

Comment (Hejnewar) [reply data/core/units/humans/Loyalist_Cavalryman.cfg:47]: It is nerf. Mountes has blade=80 so it is 20% res now.

**Review by knyghtmare — APPROVED** [review]

**Pull request merged**